src/Admin/BaseFieldDescription.php

```diff
@@ -475,14 +475,15 @@ public function setHelp($help)
         $this->help = $help;
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/admin-bundle 3.x and will be removed in version 4.0. Use Symfony Form "help" option instead.
+     *
+     * @return string
+     */
     public function getHelp()
     {
-        @trigger_error(sprintf(
```

Comment:
Currently we always add a deprecation before removing a method. Why cant we do this here ?

Reply:
Also needs template call `.getHelp('sonata_deprecation_mute')` instead of `.help`

Reply:
Done